virttest/nfs.py

```diff
@@ -162,7 +162,8 @@ def __init__(self, params):
             self.nfs_setup = True
             path.find_command("service")
             path.find_command("exportfs")
-            if distro_details.name == 'Ubuntu':
+            distro_name = distro_details.name.lower()
+            if (distro_name == 'ubuntu' or distro_name == 'redhat'):
```

Comment:
`if distro_name in ( 'ubuntu', 'redhat'):`

Reply:
@hereischen ,thanks, and fix.